test/unittests/execute_call_test.cpp

```diff
@@ -433,12 +433,10 @@ TEST(execute_call, imported_functions_call_indirect)
     ASSERT_EQ(module->codesec.size(), 2);
 
     constexpr auto sqr = [](Instance&, const Value* args, int) {
-        const auto x = as_uint32(args[0]);
-        return Value{uint64_t{x} * uint64_t{x}};
+        return Value{uint64_t{(args[0].i32)} * uint64_t{(args[0].i32)}};
```

Comment:
Why are the extra parentheses needed?

Reply:
automatic refactoring... There is "inline variable" command in CLion which placed `x` in both places where it was used.